### DISCUSSION TITLE
运行到python -m pipx ensurepath的时候

### DISCUSSION THREAD

**Discussion by JiaAiLiuzkj** [nonebot/discussions#301] [问答] [answered]
运行到python -m pipx ensurepath的时候提示You will need to open a new terminal or re-login for the PATH changes to take effect.

**Comment by zty012** [comment]
https://fanyi.youdao.com

**Comment by yanyongyu** [comment, marked as answer]
重新打开终端

**Reply by JiaAiLiuzkj** [reply]
重新打开也不行

**Reply by yanyongyu** [reply]
重新打开也不能用`nb`命令？

**Reply by JiaAiLiuzkj** [reply]
是的

**Reply by yanyongyu** [reply]
看一下你的环境变量`PATH`

**Reply by JiaAiLiuzkj** [reply]
C:\Program Files\Java\jdk1.8.0_131\bin

**Reply by yanyongyu** [reply]
环境变量path是一系列路径，不应该只有这一个

**Reply by JiaAiLiuzkj** [reply]
![image](https://user-images.githubusercontent.com/111489793/214997985-223c2f7d-4fa2-4aff-8dca-e1c3bea0ccde.png)

**Reply by yanyongyu** [reply]
你的python路径在哪里

**Reply by JiaAiLiuzkj** [reply]
![image](https://user-images.githubusercontent.com/111489793/214998324-60313a5d-697f-43cc-8d24-19f166f74562.png)

**Reply by JiaAiLiuzkj** [reply]
我记得安装py时勾选了下面的框来着

**Reply by yanyongyu** [reply]
python都不在path里你是咋用的Python命令

**Comment by JiaAiLiuzkj** [comment]
![image](https://user-images.githubusercontent.com/111489793/214998879-d80006ed-9796-45ac-87aa-e445ab547637.png)

**Reply by yanyongyu** [reply]
再运行下`pipx ensurepath`看看输出

**Reply by JiaAiLiuzkj** [reply]
![image](https://user-images.githubusercontent.com/111489793/214999977-b49ac125-7de2-4e1a-bbd9-37cd0dd878ff.png)

**Reply by yanyongyu** [reply]
关掉这个cmd再开一个 pipx install nb-cli 再试试nb

**Reply by JiaAiLiuzkj** [reply]
![image](https://user-images.githubusercontent.com/111489793/215001648-62ce490e-ab48-4754-a729-0f074839414d.png)

**Reply by yanyongyu** [reply]
`nb`呢？能不能用

**Reply by JiaAiLiuzkj** [reply]
nb可以用了

**Reply by JiaAiLiuzkj** [reply]
谢谢大佬
